### PR TITLE
Fix/seedlet bcb pullquote

### PR DESCRIPTION
**Pull request by pbking**
Extends #3575 

Changes Seedlet PullQuote block styles to match seedlet.

Minor changes to BCB to allow for configuration of citation font family and quote line heights.

Includes some margin changes for paragraph.

Before:
image

After:
image

### CODE REVIEW

File: seedlet-blocks/sass/theme.scss [hunk 1]
```diff
@@ -1,6 +1,14 @@
 @import './blocks/site-title.scss';
 @import './blocks/links.scss';
 
+/**
+ * Paragraph Margins
+ */
+
+p {
+	margin-top: 0;
```
Comment (jffng) [inline seedlet-blocks/sass/theme.scss]: This is pretty generic. Are we sure it's what we want?

For example, if I place a cover block followed by a paragraph: 

Screen Shot 2021-03-31 at 11 26 20 AM

**Comment by jffng** [comment]
The pullquote looks good :)

**Comment by pbking** [comment]
The paragraph as it existed didn't provide the same layout as before, but probably not the pr for it.  I'm still not sure how it should land for SB but I think you've demonstrated this isn't it.  I'll remove the P margins before bringing in.

File: seedlet-blocks/experimental-theme.json [hunk 2]
```diff
@@ -469,19 +475,21 @@
...
-				"fontStyle": "italic"
+				"fontSize": "32px",
+				"fontStyle": "italic",
+				"lineHeight": "1.3",
+				"fontFamily": "var(--wp--preset--font-family--headings)"
 			},
 			"border": {
-				"style": "solid",
+				"style": "none",
 				"width": "1px 0"
```
Comment (jffng) [inline seedlet-blocks/experimental-theme.json]: Nit but I think this can be removed if style is none:
```suggestion
```

**Review by jffng — APPROVED** [review]
LGTM!

**Pull request merged**